src/pages/SignInPage.jsx

```diff
@@ -15,9 +15,9 @@ const LoginPage = () => {
                 <BrandLogoIWithTextImage />
```

Comment:
img 의 크기가 디자인과 다른것같습니다 !
```suggestion
            width={238}
            height={77}

```

Reply:
직사각형인가요?

Reply:
넵 맞습니다! 아래와 같이 238 x 77 의 넓이를 가집니다!

<img width="627" alt="image" src="https://user-images.githubusercontent.com/65755120/221150161-96873423-f7da-4a92-919d-802ba4453c50.png">

Reply:
아마 그 전 로고를 사용해서 그런 것 같네요. 이 부분은 수정하겠습니다.

Reply:
이 부분 다른 이유가 실제 적용되어져있는 사이즈로 하면 로고가 찌그러지는 현상이 발생이 돼서 눈대중으로 맞는 정도로만 작성을 했습니다.